src/App.tsx

```diff
@@ -2,7 +2,7 @@ import { FC } from "react";
 
 import Navbar from "./components/atoms/Navbar";
 import Footer from "./components/organisms/Footer";
-import ProblemIndex from "./components/pages/ProblemIndex";
+import ProblemIndex from "./components/ProblemIndex";
```

Comment:
We want to keep `pages` within `components`. ;)

Reply:
For sure we want this, i don't know what happened here